run/john.conf

```diff
@@ -376,6 +376,14 @@ Frequency = 141
 # This turns off all 10 units (resulting in a timeout).
 #Config1_04A36E0FD6_0 = \xff\x03
 
+[ZTEX:sha256crypt]
+# Design tools reported possible frequency is 166.3 but tested boards
+# miss guesses, often fail unless frequency is decreased.
```

Comment:
It'd be helpful to add a similar comment for sha512crypt and Drupal7, stating those formats' design tools' reported frequency.